text/dependencies/rfcs/0000-dependency-management-overview.md

```diff
@@ -0,0 +1,197 @@
+approach, split into phases that will help to address the concerns cited in
+the Motivation section of the RFC. Due to the complex nature of the changes,
+the proposals details will be split into separate RFCs.
+
+The new system will focus on shifting all dependency-specific logic into the
+buildpacks themselves. Related automation to actually update and build the
+dependencies will be generic, so that our Github Actions can simply leverage the
+dependency-specific logic in the buildpacks. This allows for buildpack
+maintainers to take ownership on where dependencies come from, if and how
+they're compiled, and the ability to manage all related automation themselves.
+Relatedly, dependencies maintainers will be responsible for setting up general
+project-level dependency concerns like automation, best
+practices, and maintaining legacy dependencies in the dep-server.
+
+The other focus is to enable Stack flexibility by moving away from supporting
+only Ubuntu 18.04 dependencies, which is what we have now due to the way
+dependencies are compiled. While some dependencies may still need to be
+compiled or processed in some way for optimal buildpack performance, changes to
+the dependency retrieval/compilation process should allow for dependencies of
+many different OS compatibilities to be used in the buildpack.
+
+In order to implement these features in the buildpacks, there are three main
+phases:
+
+1. Federated Model Adoption In Buildpacks: [Phase 1 RFC](https://github.com/paketo-buildpacks/rfcs/blob/dependency-management-step-one/text/dependencies/rfcs/0000-dependency-management-phase-one.md)
+
+Buildpacks will adopt a federated approach to dependency management by moving
+the responsibility of dependency-specific logic to buildpacks out of the
+dep-server and related repositories.
+- Buildpacks maintainers determine if dependencies can be consumed directly from
+  upstream or must be compiled or processed in any way
+- Dependency-specific bucket is set up for metadata and compiled dependencies
+- RFC model is enacted for recording dependency rationale for every dependency
+- Remove references to Cloud Foundry compilation code, buildpacks implement
+  dependency compilation code (if needed) in Golang
+- Version retrieval code is removed from the dep-server repository and moved into Buildpacks
+- Dependencies are made available in a manner compatible with a
+  variety of stacks
+
+2. Workflow and Action Generalization: [Phase 2
+   RFC](https://github.com/paketo-buildpacks/rfcs/blob/dependency-management-step-two/text/dependencies/rfcs/0000-dependency-management-phase-two.md)
+
+All workflows and actions will be moved out of the
+[paketo-buildpacks/dep-server](https://github.com/paketo-buildpacks/dep-server/tree/main/.github)
+repository and moved into
+[paketo-buildpacks/github-config](https://github.com/paketo-buildpacks/github-config)
+with ownership by the Dependencies team. The actions and workflows be
```

Comment:
This is fair, I don't think the workflows will be too difficult to manage for Tooling maintainers, since most of the logic is going to be in the buildpack, and the workflows will just stick the steps together. I maintain Dependencies and Tooling though, so other maintainers should probably weigh in...

Reply:
I'm fine with owning the dependency workflows, as a tooling maintainer.

Reply:
Update the RFC to reflect this as we have 3/4 tooling maintainers with 👍